addons/kibana/6.8.x/kibana-2.yaml

```
@@ -0,0 +1,96 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: kibana
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: kibana
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.0-2"
```

Looks like we've had the revision version wrong on Kibana, now would be a reasonable time to get that corrected:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.2-1"
```